pom.xml

```diff
@@ -52,9 +47,11 @@
   </pluginRepositories>
 
   <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <revision>1.18</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <gitHubRepo>jenkinsci/${project.artifactId}</gitHubRepo>
     <maven.version>2.0.1</maven.version>
-    <spotbugs.skip>true</spotbugs.skip>
+    <spotbugs.threshold>High</spotbugs.threshold>
```

Comment:
There were only a few high-priority SpotBugs warnings, so I suppressed them and turned this on so that we at least discover new high-priority SpotBugs warnings.